keylime/cli/policies.py

```diff
@@ -25,21 +40,21 @@ def enforce_pcrs(tpm_policy, protected_pcrs, pcr_use):
             sys.exit(1)
 
 
-def process_allowlist(args):
-    tpm_policy = "{}"
+def process_allowlist(args: ArgsType) -> Tuple[Dict[str, Any], Optional[str], str, str, Dict[str, Any]]:
+    tpm_policy_str = "{}"
     ima_policy_name = ""
     mb_refstate = None
-    ima_sign_verification_keys = []
+    ima_sign_verification_keys = ""
```

Comment:
Seems that this is a JSON string, and array. IMHO the correct default is `ima_sign_verification_keys = "[]"`